api/proto/teleport/legacy/types/webauthn/webauthn.proto

```diff
@@ -41,23 +41,6 @@ option (gogoproto.unmarshaler_all) = true;
 // WebAuthn messages used by server storage.
 // -----------------------------------------------------------------------------
 
-// SessionData stored by the Relying Party during authentication ceremonies.
-// Mirrors https://pkg.go.dev/github.com/go-webauthn/webauthn/webauthn#SessionData.
-message SessionData {
-  // Raw challenge used for the ceremony.
```

Comment:
Is it okay to remove a proto message like this? Are there backwards compatibility concerns?

Reply:
This proto message was only being used as a Go struct - https://github.com/gravitational/teleport/pull/36588#discussion_r1452436992, so there shouldn't be any backwards compatibility concerns.